be/src/util/threadpool.h

```diff
@@ -336,6 +336,8 @@ class ThreadPool {
     // Protected by _lock.
     std::unordered_set<Thread*> _threads;
 
+    std::unordered_set<pthread_t> _pthread_ids;
```

Comment:
What's the relationship between `_threads` and `_pthread_is`?

Reply:
if join threads in thread_pool, the thread may be deleted by pthread, so finally join pthread, collected in _pthread_ids.